src/style.css

```diff
@@ -1,3 +1,72 @@
 body {
   margin: 0;
```

Comment:
Avoid using tag names for styling (except `html` and `body`). This can lead to unexpected results if new elements of the same type are added to the page. It would be better to add a class to the body tag and style it. (Checklist: Don't use tag names for styling)